util/reggen/fpv_csr.sv.tpl

```diff
@@ -38,6 +37,15 @@ module ${block.name}_csr_assert_fpv import tlul_pkg::*; import ${block.name}_reg
 
 `ifdef UVM
   import uvm_pkg::*;
+  % if block.name == "alert_handler":
+  `define REGWEN_PATH dut.i_reg_wrap.u_reg
+  % elif block.name == "flash_ctrl":
+  `define REGWEN_PATH dut.u_flash_ctrl.u_reg
+  % else:
+  `define REGWEN_PATH dut.u_reg
+  % endif
```

Comment:
somehow my comment isn't disappear.
suggest to use `hier_path` from hjson file and if TB adds extra level, like flash_ctrl, we can define the path in tb

Reply:
That is a good point! Thanks weicai, i will update the code as you recommended.